app_dart/lib/src/request_handling/authentication.dart

```diff
@@ -60,6 +60,7 @@ import 'exceptions.dart';
 /// See also:
 ///
 ///  * <https://cloud.google.com/appengine/docs/standard/python/reference/request-response-headers>
+// TODO(chillers): Remove (1) when DeviceLab has migrated to LUCI, https://github.com/flutter/flutter/projects/151#card-47536851
```

Comment:
With the same logic being implemented in `SwarmingAuthenticationProvider`, can we delete this part now?

Reply:
`/api/append-log`, `/api/get-authentication-status`, and `'/api/update-agent-health'` still use the `AuthenticationProvider`, so not yet. I'll investigate what endpoints require what auth methods, and update in a separate PR.

My expectation is for this to be done after the devicelab to LUCI migration as we will be able to remove several request handlers.

Reply:
SGTM